build/emoji/emoji_gen.mjs

```diff
@@ -141,30 +143,92 @@ function genSkinVariations(emoji, index, nextOrder) {
         variation.category = emoji.category;
         variation.subcategory = emoji.subcategory;
```

Comment:
this is removed in the trimPropertiesFromEmoji, so no need to add it.

Reply:
✅  Updated as per request/discussion